core/transaction/transaction.go

```diff
@@ -327,6 +326,17 @@ func (tx *Transaction) GetProgramHashes() ([]Uint160, error) {
 	case TransferAsset:
 	case Record:
 	case BookKeeper:
+		issuer := tx.Payload.(*payload.BookKeeper).Issuer
+		signatureRedeemScript, err := contract.CreateSignatureRedeemScript(issuer)
```

Comment:
'fallthrough' could be used to avoid duplicated code, please fix other transaction type together.
- RegisterAsset
- DataFile
- BookKeeper
- PrivacyPayload